cypress/integration/content/headTags_spec.js

```diff
@@ -6,7 +6,11 @@ describe("Head Tags", () => {
     cy.goHome();
   });
   it("creates and deletes new head tag", () => {
-    cy.get("#MainNavigation").contains("All Field Types").click();
+    cy.get("#MainNavigation")
```

Comment:
When testing functionality we should load the url directly to that area, if possible. Instead of navigating the app links